app/Models/License.php

```diff
@@ -175,12 +176,24 @@ public static function adjustSeatCount($license, $oldSeats, $newSeats)
             return true;
         }
         // Else we're adding seats.
-        DB::transaction(function () use ($license, $oldSeats, $newSeats) {
```

Comment:
I'm okay with pulling this transaction, but we should explain that we're pulling this transaction and why. It does seem like a pretty reasonable failsafe to have.

Reply:
I moved the transactions into the Collection chunks. Having one BIG transaction for a boatload of new LicenseSeats is probably going to blow up any MySQL server.